src/cli.js

```diff
@@ -2,7 +2,7 @@ import path from 'path';
 
 import argv from 'yargs';
 
-import { version } from 'json!../package';
```

Comment:
Ah, this doesn't work anymore in webpack2, so I changed it to the standard way which works.